QuickDeployment/V2/QuickDeploymentDesktopReadme.md

```diff
@@ -0,0 +1,360 @@
+We recommend you leave the VM running while it's happening; you can keep tabs on its progress by opening Nexus to the ChocolateyInternal repository and checking the available packages.
+
+---
+
+## Step 3: Review Hosted Services
+
+### Nexus Repository
+
+> :memo: **Note**
+>
+> If you have changed the hostname from the default, or you're using a FQDN for the server setup, this hostname will have changed.
+> In that case, substitute the hostname for the new FQDN.
+
+|            URL             |  Login  |  Initial Password   |      API Key      |
+| :------------------------: | :-----: | :-----------------: | :---------------: |
+| <https://chocoserver:8443> | `admin` | `{{NexusPassword}}` | `{{NugetApiKey}}` |
+
+If you have not setup QDE as internet-enabled, Nexus will be initially configured with anonymous authentication enabled.
+On first logging in to the admin account, you will be prompted to change your password and then select whether or not to enable anonymous authentication going forward.
+If you intend to set up QDE as an internet-facing server, we recommend you keep anonymous authentication **OFF**.
+If your QDE instance will **not** be internet-facing and only used over a VPN or other secured network, it is generally safe to leave anonymous authentication turned on.
+
+Unless security requirements in your organization stipulate that role-based access controls be in place, this is our recommended configuration for QDE in its default configuration.
+
+Sources configured in Chocolatey can only read data from their remote endpoints, and cannot delete items.
+If you need to limit the packages people have access to, control this with separate Hosted and Group repositories.
+Consult the [Nexus documentation](https://help.sonatype.com/repomanager3) or reach out to Chocolatey Support for more information.
+
+### Jenkins
+
+|           URL           |  Login  |   Initial Password    |
+| :---------------------: | :-----: | :-------------------: |
+| <http://localhost:8080> | `admin` | `{{JenkinsPassword}}` |
+
+For using Jenkins, please refer to our [online documentation](https://chocolatey.org/docs/how-to-setup-internal-package-repository).
+
+At most, you will need to:
+
+1. Login to Jenkins and complete first-time setup (see below).
+1. Change your password (by going to `People > admin > Configure` on the sidebar, scroll down to change password section).
+1. Enable the pre-configured jobs to run on the schedule of your choosing.
+   Our documentation can assist with ensuring this is done correctly.
+
+<details>
+    <summary><strong>First-Time Setup</strong></summary>
+
+In order to begin using Jenkins, the following steps will need to be performed:
+
+1. Visit <http://localhost:8080> in a web browser
+2. Enter the administrator password into the textbox and click `Continue`
+3. From the Plugin Installation Page select "Install Suggested Plugins"
+   > :memo: **Note**
+   >
+   > Some of these may fail to install, and that is OK.
```

Comment:
not sure how well our Markdown will handle this. Typically you want a line break between the start and ends of these.